react/queries/orderForm.gql

```diff
@@ -1,6 +1,7 @@
 query orderForm {
   orderForm {
     cacheId
+    sellers
```

Comment:
this property doesn't exist in the [orderForm schema](https://github.com/vtex-apps/store-graphql/blob/master/graphql/types/OrderForm.graphql#L1-L26).

Reply:
@lucasayb this field is not a scalar, can you specify which sub-fields do you want to query from it?